### ISSUE TITLE
Error on SAP installation

### ISSUE THREAD

**Issue by HasmineRoldan** [brandoncaulfield/sap-nw-abap-trial-docker-windows#5]
INFO 2020-10-13 ee:14:26.242 (root/sapinst) (startInstallation) [/bas/749 REL/bc_749 REL/src/ins/SAPINST/impl/src/syslib/process/syuxccuren.cpp:546) id=syslib.pro cess.currentProcess Env.cwdSuccessful csyCurrent Process Environment impi::setWorkingDirectory(const cSypathe /tmp/sapinst_instdir/WW73/SBC/STANDARD)
Working directory changed to /tmp/sapinst_instdir/NW73/SBC/STANDARD

####################################
Abort execution because of
Step returns modlib.jslib.caught exception
####################################

Installation of NPL failed, please check log files in /tmp/sapinst_instdir for possible root cause.

![2020-10-13 (2)](https://user-images.githubusercontent.com/65996518/96028945-8be23480-0e8c-11eb-8ad5-f357df69b3d3.png)

**Comment by brandoncaulfield** [comment]
Hi Hasmine, I think this exception has to do with the password you at during the installation. It was also raised in another issue here but the solution was to re run the installation and make sure the password is 8 characters or longer.

Try that and let us know?

**Comment by brandoncaulfield** [comment]
I'm going to close this issue for now. If you're still running into it please let me know :)

**Comment by HasmineRoldan** [comment]
Hi Brandon, I tried reinstalling it again and used 9 character password but I'm still having the same error.

**Comment by brandoncaulfield** [comment]
Hi @HasmineRoldan , that is strange. Try have a look at the article I posted here - https://blogs.sap.com/2020/08/04/sap-as-abap-7.52-sp04-trial-dev-edition-issues-and-lessons/

The only thing I can think of is trying a completely fresh install from the very beginning (deletion he current docker image) and try8g that 8+ character password. 


If that still fails try raising an issue in the comments with Julie Plummer here - https://blogs.sap.com/2019/07/01/as-abap-752-sp04-developer-edition-to-download/

Keep us posted!

**Comment by HasmineRoldan** [comment]
Hi, @brandoncaulfield  I just found out that the ASE license on the SAP Trial that I downloaded is already expired that causes the error on the installation. I will just try to download the latest SAP Trial which is 7.52 SP04 and try to install it again, hope it works. Thanks for helping me!
![2020-10-21 (2)](https://user-images.githubusercontent.com/65996518/96619590-f1c13700-1338-11eb-9d4c-f5d3813b58b0.png)
